test/e2e/testdata/should-preserve-keywords/directory/emailTemplates/welcome_email.json

```diff
@@ -0,0 +1,11 @@
+{
+    "template": "welcome_email",
+    "body": "./welcome_email.html",
+    "from": "",
+    "resultUrl": "https://##DOMAIN##/welcome",
+    "subject": "Welcome",
+    "syntax": "liquid",
+    "urlLifetimeInSeconds": 3600,
+    "enabled": false
+  }
```

Comment:
This file was missing and is necessary to create an additional assertion for E2E tests.